### PR TITLE
Fix typos and update details for Ubuntu installation instruction

### PR DESCRIPTION
**Pull request by msylvestre**
Fix the repo URL
Fix the ruby version
Added detail how to setup MySQL
Added detail about setuping the Google Auth
Added detail if you run peatio form a server and not from a local machine
Added info about the "working trade UI"

### CODE REVIEW

File: docs/setup-ubuntu.md [hunk 1]
```diff
@@ -221,17 +238,44 @@ bundle exec rake solvency:liability_proof
...
+
+
 Once server is up and running, **visit [http://localhost:3000](http://localhost:3000)**
 
-Sign in:
+Sign in with Google SSO
 
-* user: admin@peatio.dev
-* pass: Pass@word8
+NOTE : Looks like the trading/MArket UI is broken.  Install an run https://github.com/rubykube/peatio-trading-ui in parallel of peatio.
```
Comment (yivo) [inline docs/setup-ubuntu.md]: @msylvestre I don't think we can merge this. Trading UI is fully working. Check https://github.com/rubykube/peatio/blob/master/docs/releases/1.5.0.md

**Review by yivo — CHANGES_REQUESTED** [review]

File: docs/setup-ubuntu.md (hunk 1 above)
Comment (msylvestre) [reply docs/setup-ubuntu.md]: Hi yivo, What I mean is that _now_ you need to install trading-ui as a separate server, you can't just run peatio "as-is" as it was originally written in the installation instruction.

But I understand, your point, it's not clear, let me me rephrase.  I'm also understanding better "why and how", after chatting more with the folk on Telegram !

Comment (yivo) [reply docs/setup-ubuntu.md]: @msylvestre Hi
Could you just pul link to 1.5 release notes or to trading-ui README file instead of this paragraph?

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by yivo** [comment]
@msylvestre Thank you

**Pull request merged**